packages/garbo/src/fights.ts

```diff
@@ -1928,6 +1929,36 @@ const freeKillSources = [
     },
   ),
 
+  new FreeFight(
+    () =>
+      globalOptions.ascend &&
+      0.1 * garboValue($item`spice melange`) * itemDropModifier() >
```

Comment:
use either libram's `getModifier` or mafia's `numericModifier` rather than `itemDropModifier`

Also this will always be an extremely huge number